resolwe/flow/executors/constants.py

```diff
@@ -9,7 +9,7 @@
 SETTINGS_VOLUME = Path("/settings")
 SOCKETS_VOLUME = Path("/sockets")
 
-CONTAINER_TIMEOUT = 300
+CONTAINER_TIMEOUT = 600
```

Comment:
What is the reasoning behind this change? I assume that 5 minutes should be more than enough for the container to start.

Reply:
It was an actual event in kuberenetes when processing container took 20s over 5 minutes to start.
I believe the cause was downolading docker image but I am not sure.

Reply:
Ok, it explains the problem. In Docker executor, the process is divided into two parts, pulling the image and starting the container, where only the second part has a timeout. Is it possible to do something similar in Kubernetes?